Lab.md

```diff
@@ -46,27 +42,29 @@ Before moving on, install additional NuGet packages that you will use later.
 Select **Tools > Nuget Package Manager > Package Manager Console**. In the console window, run the following commands:
 ```powershell
 Install-Package "Microsoft Graph"
-Install-Package "Microsoft.Identity.Client" -pre
+Install-Package "Microsoft.Identity.Client"
 Install-Package "System.Configuration.ConfigurationManager"
 ```
 
-## Exercise 2: Register a native application with the Application Registration Portal
+## Exercise 2: Register a native application with the Azure portal
 In this exercise, you will create a new Azure AD native application using the Application Registry Portal.
-1. Open a browser and navigate to the [Application Registration Portal](). Login using a **personal account**(aka: Microsoft Account) or **Work or School Account**
-2. Select **Add an app** at the top of the page
-3. On the Register your application page, set the **Applcaiton Name** to **Calendar** and select **Create**
-4. On the **Calendar Registration** page, under the **Properties** section, copy the **Application Id** as you will need it later
-5. Select **Add Platform** button on the registration page. Choose **Native Application** in the dialog box.
-6. Once completed, move to the bottom of the page and select **Save**
+
+1. Open a browser and navigate to the [Azure portal app registrations page](https://aka.ms/AppRegistrations). Login using a **personal account** (aka: Microsoft Account) or **Work or School Account**.
+2. Select **New registration** at the top of the page.
+3. On the Register an application page, set the **Application Name** to **Calendar**. Under "Supported account types", choose **Accounts in any organizational directory and personal Microsoft accounts**.
+4. Select **Register** at the bottom of the page.
+5. On the **Calendar** page, copy the **Application (client) ID** as you will need it later.
+6. Go to the **Authentication** page. Under **Suggested Redirect URIs for public clients (mobile, desktop)** check the boxes next to **urn:ietf:wg:oauth:2.0:oob**  and **https://login.microsoftonline.com/common/oauth2/nativeclient**.

```

Comment:
I believe that  **urn:ietf:wg:oauth:2.0:oob** is no longer recommended for security reasons. On the other hand this is the default for MSAL.NET ...